less/index.less

```diff
@@ -54,6 +57,149 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	// background-color:red;
 }
 
+html{
+	font-size:10px;
+	background-color:@back;
+}
+@back:lightgrey;
+.mainColor(){
+	background-color:azure;
+	border:3px solid black;
+}
+
+.giantContainer{
+	max-width:90%;
+	height:auto;
+	margin-left:10%;
+	margin-right:10%;
+}
+h2, h3{
+	margin-bottom:7%;
+	font-size:2.75rem;
+	font-weight:bold;
+}
+.topContainer{
+	border-radius: 25px;
+	max-width:90%;
+	height:auto;
+	margin-left:10%;
+	margin-right:10%;
+	display:flex;
+	.mainColor();
+	.profile{
+		max-width:30%;
+		padding:3%;
+		height:auto;
+	}
+	.topContent{
+		display:flex;
+		flex-direction:column;
+
+	}
+	h1{
+		// margin-top:3%;
+		// margin-left:5%;
+		// margin-bottom:5%;
+		// width:100%;
+		text-align:center;
+		font-size: 5rem;
+		// border-bottom:3px dotted darkslategrey;
+		// height:12%;
+		
+	}
+	li{
+		font-size:2.5rem;
+		margin:6%;
+		padding-left:8%;
+	}
+}
+
+
+
+nav{
+	display:flex;
+	justify-content:center;
+	a{
+		padding:3%;
+		text-decoration:none;
+		color:slategrey;
+		font-size:2.5rem
+		
+	}
+	
+	a:hover {
+		border: 3px dotted #3a7999;
+		color: #3a7999;
+		background: rgba(0,0,0,0);
+	}
+
+}
+p{
+	font-size:2.25rem;
+}
+.midMargin(){
+	margin:2%;
+	padding:2%;
+	text-align:center;
+	border:6px double ghostwhite;
+	border-radius: 25px;
+
+}
+.midContainer{
+	 .ed{
+		.midMargin;
```

Comment:
We talked about this already, but you're repeating this `max-width: 30%` a lot and it could be done in a cleaner way